### DISCUSSION TITLE
Is the health behaving strange?

### DISCUSSION THREAD

**Discussion by chinghei1007** [ppy/osu#13471] [Q&A] [answered]
Is it my fault or, is the lazer harder to pass a map?
I felt a dramatic decrease in HP comparing to the classic osu!
I could pass the same map every single time I played it on classic
But I would fail the map every single time on lazer (on the same difficulty)
I played like 10 5* maps and I would fail all of them on lazer but not classic
I tried redownloading the maps instead of direct import
but it didn't help ;^;
Any thoughts?
I'm using the most recent version of lazer

**Comment by PercyDan54** [comment, marked as answer]
Health drain is working differently in lazer now.

**Reply by bdach** [reply]
To quote the relevant portion of the README in particular:

> IMPORTANT: Gameplay mechanics (and other features which you may have come to know and love) are in a constant state of flux. Game balance and final quality-of-life passes come at the end of development, preceded by experimentation and changes which may potentially reduce playability or usability. This is done in order to allow us to move forward as developers and designers more efficiently. If this offends you, please consider sticking to the stable releases of osu! (found on the website). We are not yet open to heated discussion over game mechanics and will not be using github as a forum for such discussions just yet.
